tools/deployment/make_linux_package.sh

```diff
@@ -133,7 +192,7 @@ function main() {
   rm -f $OUTPUT_PKG_PATH
   mkdir -p $INSTALL_PREFIX
 
-  log "copying osquery binaries"
+  log "copying osquery binaries to $INSTALL_PREFIX"
```

Comment:
In theory the manual copying can be replaced with `make DESTDIR="${INSTALL_PREFIX}" install`. This should also take care of any pre/post-install step

Reply:
Agree, I'll follow up with that change in #5842